### PR TITLE
README.md: document kola test writing

### PR DESCRIPTION
**Pull request by pbx0**

### CODE REVIEW

**Comment by philips** [comment]
cc @robszumski it would be great to pull this into the SDK docs. This is something where the community could help out.

**Comment by marineam** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -15,19 +15,63 @@ like Apache's directory index for your software download repository.
...
+### kola test writing
+A kola test is a go function that is passed a `platform.TestCluster` to
+run code against.  Its signature is `func(platform.TestCluster) error`
+and must be registered and built into the kola binary. 
+
+A `TestCluster` implements the `platform.Cluster` interface and will
+give you access to a running cluster of CoreOS machines. A test writer
+can interact with these machines through this interface.
+
+To see test examples look under kola/tests in the mantle codebase.
```
Comment (philips) [inline README.md]: Linkify? https://github.com/coreos/mantle/tree/master/kola

Comment (pbx0) [inline README.md]: done

**Pull request merged**